src/OrleansConsulUtils/OrleansConsulUtils.csproj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Orleans.Core.Abstractions\Orleans.Core.Abstractions.csproj" />
```

Comment:
Please leverage transitive P2P references, need to reference only OrleansRuntime.

Reply:
hmm transitive P2P references somehow doesn't work